cmd/main.go

```diff
@@ -19,10 +22,10 @@ import (
 )
 
 const (
-	get    = "GET"
-	put    = "PUT"
-	post   = "POST"
-	delete = "DELETE"
+	getCode    = "GET"
```

Comment:
```suggestion
	codeGet    = "GET"
```

rename the other too